bin/transform

```diff
@@ -95,11 +97,61 @@ def parse_geographic_columns(gisaid_data: pd.DataFrame) -> pd.DataFrame:
     Expands the string found in the column named `location` in the given
     *df*, creating four new columns. Returns the modified ``pd.DataFrame``.
     """
+    def titlecase(text: Union[str, pd._libs.missing.NAType],
+        articles: Set[str] = {}, abbrev: Set[str] = {}) -> Optional[str]:
+        """
+        Returns a title cased location name from the given location name
+        *tokens*. Ensures that no tokens contained in the *whitelist_tokens* are
+        converted to title case.
+
+        >>> articles = {'a', 'and', 'of', 'the', 'le'}
+        >>> abbrev = {'USA', 'DC'}
+
+        >>> titlecase("the night OF THE LIVING DEAD", articles)
+        'The Night of the Living Dead'
+
+        >>> titlecase("BRAINE-LE-COMTE, FRANCE", articles)
+        'Braine-le-Comte, France'
+
+        >>> titlecase("auvergne-RHÔNE-alpes", articles)
+        'Auvergne-Rhône-Alpes'
+
+        >>> titlecase("washington DC, usa", articles, abbrev)
+        'Washington DC, USA'
+        """
+        if not isinstance(text, str):
+            return
+
+        words = enumerate(regex.split(r'\b', text, flags=regex.V1))
+
+        def changecase(index, word):
+            casefold = word.casefold()
+            upper = word.upper()
+
+            if upper in abbrev:
+                return upper
+            elif casefold in articles and index != 1:
+                return word.lower()
+            else:
+                return word.title()
+
+        return ''.join(changecase(i, w) for i, w in words)
+
     geographic_data = gisaid_data['location'].str.split(r'\s*/\s*', expand=True)
     location_columns = ['region', 'country', 'division', 'location']
 
+    # Manually curate set of tokens that should not be cast to title case
+    articles = {
+        'de', 'del', 'di', 'do', 'l', 'la', 'le', 'las', 'los', 'nad', 'of',
+        'op', 'the', 'y'
```

Comment:
> @kairstenfay Thank you for tackling the casing problem!
> 
> Something I noticed while testing is that the casing in the `annotations.tsv` can vary as well. It's outside the scope of this PR, but just wanted to point it out. (It might be something to add as a check in #54?)

Oh, good catch. I didn't realize annotations needed title casing as well. It's not necessarily outside the scope of this PR if we move the title casing form `parse_geographic_columns()` to a new function after `update_metadata()`. Maybe the function could be named `titlecase_columns()` and accept a list of columns that should be title cased. Perhaps then we could even extend this to other columns like author names, etc. Thoughts?

Reply:
+1 on des and en. 
I see "sur" occurs commonly in Francophone countries (Montignies-Sur-Sambre). It's also its own word meaning "south" in Spanish that may show up in some locations where it ought to be capitalized. Given "sur"s prevalence as an article in Francophone locations, though, I think it would be fair to manually annotate Hispanic locations.